addons/website_blog/migrations/9.0.1.0/post-migration.py

```diff
@@ -27,11 +27,10 @@ def convert_blog_post_cover(env):
         post = post_obj.browse(row[0])
         post.cover_properties = (
             '{{"background-image": "url({})",'
-            ' "opacity": "1",'
+            ' "opacity": "0",'
```

Comment:
If this doesn't have `opacity`, then the image won't be shown. This was copied from the style given by Odoo.